app/models/project.server.ts

```diff
@@ -48,6 +50,51 @@ export class SearchProjectsError extends Error {
   message = "There was an error while searching for projects."
 }
 
+export function isProjectTeamMember(profileId: string, project: ProjectComplete) {
+  const isProjectMember = project?.projectMembers.some((p) => p.profileId === profileId);
+  const isProjectOwner = profileId === project?.ownerId;
+  return isProjectMember || isProjectOwner;
+}
+
+export function getProjectTeamMember(profileId: string, project: ProjectComplete) {
+  return project?.projectMembers.find(p => p.profileId === profileId);
+}
+
+export async function getProject({
+  id
+}: Pick<Projects, "id">) {
+  return await db.projects.findFirst({
+    where: { id },
+    include: {
+      skills: true,
+      disciplines: true,
+      labels: true,
+      projectStatus: true,
+      owner: true,
+      projectMembers: {
+        include: {
+          profile: { select: { firstName: true, lastName: true, email: true } },
+          contributorPath: true,
+          practicedSkills: true,
+          //role: true,
```

Comment:
I suggest delete commented code

Reply:
I think this is instead an error that we haven't updated our schema and migrations from latest changes from the project-lab repo. I will work on this next.